lib/ts/recipe/session/utils.ts

```diff
@@ -104,6 +111,21 @@ export function validateAndNormaliseUserInput(
     appInfo: NormalisedAppinfo,
     config?: TypeInput
 ): TypeNormalisedInput {
+    let inputValidation = validate(config, InputSchema);
+    if (inputValidation.errors.length > 0) {
+        let path = inputValidation.errors[0].path.join(".");
+        if (path !== "") {
+            path += " ";
+        }
+        let errorMessage = `${path}${inputValidation.errors[0].message}`;
+        throw new STError(
+            {
+                type: STError.GENERAL_ERROR,
+                payload: new Error(`Supertokens Recipe Config Schema Error: ${errorMessage}`),
+            },
+            recipeInstance.getRecipeId()
+        );
+    }
```

Comment:
This code is duplicated from the other recipe.. Move this code to utils in the ts/ folder